esmvaltool/recipes/recipe_autoassess_landsurface_permafrost.yml

```diff
@@ -5,9 +5,6 @@ documentation:
   description: |
     Recipe that runs the Autoassess Land-surface assessment area diagnostic.
     Land surface area and ice masks are time independent.
-    Climatological files are stored externally to avoid overloading the
-    ESMValTool source. See /gws/nopw/j04/esmeval/autoassess_specific_files
-    (on JASMIN).
```

Comment:
maybe not just fully remove this but tweak it like @remi-kazeroni mentions, saying "dude, you needs these files, currently you have just a few options get them - ie JASMIN"

Reply:
But the clim files are *not* needed by permafrost, so removal of this line is correct. See https://github.com/ESMValGroup/ESMValTool/issues/3065#issuecomment-1466643724

Reply:
yep, my bad! Sorry buds